Sources/WalletConnectModal/Modal/Screens/WalletList.swift

```diff
@@ -116,20 +118,22 @@ struct WalletList: View {
     func viewAllItem() -> some View {
         VStack {
             VStack(spacing: 3) {
-                let startingIndex = (2 * numberOfColumns - 1)
+                let viewAllWalletsFirstRow = wallets.dropFirst(2 * numberOfColumns - 1).prefix(2)
```

Comment:
Is it safe? prefix is crashing app if out of index as I remember

Reply:
It's crashing when the value to prefix ends up being negative, it's okay if you prefix higher number of elements than there is in the array. So in this case it's safe.